schema_editor/app/scripts/views/recordtype/detail-add-edit-partial.html

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
This looks like an almost exact copy of the add-edit-partial. Would it be worth it to try and combine them?